### DISCUSSION TITLE
组件库构建和产物优化讨论

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#102] [General]
1. 引入方式（全局、单独）
2. 指令命名统一
3. 打包方式
4. 依赖问题
5. feature组件类型声明文件的生成与导出
6. feature样式/size/i18n添加通用设置项的顶层注入组件

**Comment by TinsFox** [comment]
对ssr框架的支持可能需要有特别的处理，在react中 next框架对css的引入是有限制的

**Comment by GaoNeng-wWw** [comment]
现阶段组件库还没有类型提示，这个问题还挺致命的
